testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViewsLegacy.java

```diff
@@ -150,6 +150,26 @@ public void testCurrentUser()
                 .hasMessageContaining("Failed parsing stored view 'hive.default.current_user_hive_view': line 1:20: mismatched input '('");
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testRunAsInvoker()
```

Comment:
should this go into `AbstractTestHiveViews` ?

Reply:
If I understand correctly, it makes sense only if legacy hive views are used.
I am not an expert here though so I may be missing something.

Reply:
Is this relevant?

https://github.com/trinodb/trino/blob/6ceec25c5d4cdf3ba7e0c2b291f98a56b71b27b1/plugin/trino-hive/src/main/java/io/trino/plugin/hive/ViewReaderUtil.java#L88-L95

Reply:
I added a siimlar test in `TestHiveViews`, since the one from legacy would not work.

Reply:
Why wouldn't it? what's the difference?

Reply:
Views created in hive won't work. We need to create one using Trino.
BTW your comment reaction time is absurdly good.